frontend/app/sysupport-admin/index/template.hbs

```diff
@@ -1,19 +1,21 @@
 <h2>{{t 'sysupport.admin.projects'}}</h2>
 
 <div class="uk-overflow-auto" uk-scrollspy="target: > div; cls:uk-animation-slide-bottom-small; delay: 50;">
-    {{#cc-data-table headers=headers model=model as |results|}}
+  {{#cc-data-table headers=headers model=model as |results|}}
     {{#each results as |project index|}}
-      <tr>
-      {{#link-to "sysupport-admin.detail" project.id data-test-project=index class="uk-link-reset"}}
-        <td>{{project.name}}</td>
-      {{/link-to}}
+    {{#link-to "sysupport-admin.detail" project.id data-test-project=index class=(if (gte alertTime project.totalTime._milliseconds) "uk-text-danger uk-link-reset" "uk-link-reset") tagName="tr"}}
```

Comment:
`if (gte alertTime project.totalTime._milliseconds`: instead of doing this here, please add a computed property on the project model.
this would make the code more readable

Reply:
and also it would be fancy if you could use the click-cursor on hover to indicate a clickable item.